resources.py

```diff
@@ -910,6 +910,53 @@ def enable_dhcp_in_networkd(self, interface):
         # Wait a few seconds for the DHCP to be applied
         time.sleep(5)
 
+    def prepare_nested_virtualization(self):
```

Comment:
Functions should only be part of `resources.py`, when it would make the tests too complicated if they were included in the test functions directly, distracting from what we actually want to test.

The idea with the `test_` files is: A customer can read them and understand what we are doing, maybe without having to understand complicated details. A bit of an exception are LBaaS tests, as they would get very long and hard to understand, if we included the whole LBaaS setup with each test.

I don't think your functions here qualify: You should inline your code in the test functions, to demonstrate what we are checking / what the platform is meant to be capable of.